src/components/ConversationSettings/LinkShareSettings.vue

```diff
@@ -123,6 +125,7 @@ export default {
 			showPasswordField: false,
 			isSaving: false,
 			isSendingInvitations: false,
+			isValid: true,
```

Comment:
Also should be done/checked in `NewConversationSetupPage`, as most passwords should be provided from the beginning

And in this component, `handleSetNewPassword` method should be blocked with invalid password, so keydown 'Enter' won't send it